.github/workflows/pr-testing-with-test-project.yml

```
@@ -17,5 +17,5 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Run test
-        run: NODE_IMAGE_TAG=${{ matrix.node }} docker-compose up --abort-on-container-exit --remove-orphans --force-recreate
+        run: NODE_IMAGE_TAG=${{ matrix.node }} docker compose build --abort-on-container-exit --remove-orphans --force-recreate
```

```suggestion
        run: NODE_IMAGE_TAG=${{ matrix.node }} docker compose up --abort-on-container-exit --remove-orphans --force-recreate
```

let's try this according to https://docs.docker.com/compose/migrate/#unsupported-in-v2